products.json

```diff
@@ -0,0 +1,351 @@
+[
+  {
+    "_id": {
+      "$oid": "663cd8bbda2083950a8b181d"
+    },
+    "categoryId": 1,
+    "subcategoryId": 1,
+    "description": "This is an example product description.",
+    "detailedDescription": [],
+    "price": 225,
+    "quantity": 11,
+    "discount": 5,
+    "seller": 2,
+    "brand": "ZTE",
+    "state": 1,
```

Comment:
Why don't you store them in the DB?

Reply:
Now it is a correct version. 14 files should be. Sorry.